controllers/articles.js

```diff
@@ -96,73 +113,101 @@ const articles = {
    * Deleting a post based to its slug.
    *
    * For avoiding permanent deletion of the content from the database
-   * we set the flag to the article that it is deletes.
+   * we set the flag to the article that it is deleted.
    */
   deleteArticle: async (req, res) => {
     const { slug } = req.params;
     const row = await article.update({ deleted: 1 }, { where: { slug, deleted: 0 } });
     if (row[0] === 0) {
       return res.status(404).send({
         status: res.statusCode,
-        message: 'No article found for this slug',
+        message: 'No article found for this slug'
       });
     }
     return res.status(200).send({
       status: res.statusCode,
-      message: 'Article deleted successfully',
+      message: 'Article deleted successfully'
     });
   },
 
   /*
-  * Retieving all the published articles based to the author
-  * and the status of the article (Published).
-  */
+   * Retrieving all the published articles based to the author
+   * and the status of the article (Published).
+   */
   getAllPublishedArticles: async (req, res) => {
     try {
       const { id } = req.user;
-      const authorInfo = await User.findOne({ where: { id }, attributes: ['username', 'bio', 'image', 'following'] });
+      const authorInfo = await User.findOne({
+        where: { id },
+        attributes: ['username', 'bio', 'image', 'following']
+      });
       const response = await article.findAll({
         where: {
           author: id,
-          status: 'Published',
+          status: 1,
```

Comment:
@avpaul for a better description of the status, we can still be using published but we change all to small laters.